src/game/Game.kt

```diff
@@ -11,23 +11,23 @@ class Game(private val player: Player, private val worldMap: WorldMap) {
     private var timeCounter: Int = 0
 
     fun run() {
-        while(!Thread.interrupted()) {
-            player.actor.loseOneLifeUnit()
-            when (player.chooseNextMove()) {
-                Action.MOVE_NORTH -> moveNorthLogic()
-            }
-            if (!player.actor.alive || worldMap.getNode(worldMap.positionFor(player.actor)) is ExitNode) {
-                break
+        try {
```

Comment:
try catch should be inside the while loop, otherwise any exception terminates the game